sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -206,15 +206,20 @@ public virtual TokenCredential Credential
                 }
                 else
                 {
-                    _credential = new ClientSecretCredential(
-                        GetVariable("TENANT_ID"),
-                        GetVariable("CLIENT_ID"),
-                        GetVariable("CLIENT_SECRET"),
-                        new ClientSecretCredentialOptions()
-                        {
-                             AuthorityHost = new Uri(AuthorityHostUrl)
-                        }
-                    );
+                    var clientSecret = ClientSecret;
```

Comment:
Should we set something explicit in the Test Resources output and key on that?   I know this will work, but it isn't obvious for those who don't know how it works.

Reply:
Yeah I considered it but didn't seem to add much value.